tests/app/main/helpers/test_frameworks.py

```diff
@@ -0,0 +1,294 @@
+# -*- coding: utf-8 -*-
+import pytest
+from nose.tools import assert_equal
+from app.main.helpers.frameworks import get_statuses_for_lot
+
+
+def get_lot_status_examples():
+    # limit, drafts, complete, declaration, framework
+    cases = [
+        # Lots with limit of one service
+        (
+            [True, 0, 0, None, 'open'],
+            []
+        ),
+        (
+            [True, 1, 0, None, 'open'],
+            [{
+                'title': u'Started but not complete',
+                'type': u'quiet'
+            }]
+        ),
+        (
+            [True, 0, 1, None, 'open'],
+            [{
+                'title': u'Marked as complete',
```

Comment:
We have this:
![image](https://cloud.githubusercontent.com/assets/355079/11398927/c629ff2e-937b-11e5-87fc-2a732c213dfc.png)

I assumed there was already a banner on this page specifically if they hadn’t done the declaration… doesn’t seem to be the case so I’ll add that.